cdf-core/cdf/js-modules/dashboard/Dashboard.lifecycle.js

```diff
@@ -506,8 +534,12 @@ define(['./Dashboard', '../Logger', 'amd!../lib/underscore', '../components/Unma
     },
 
     /**
+     * Updates a specific component
```

Comment:
Is this correct? I got the idea that this method gathers components to update (as the comment inside the method describes), and then calls updateAll with all the gathered components.
UpdateAll in turn readies a queue for updating, and then calls updateComponent on each component, this final one being the actual method which updates a specific component